packages/cozy-harvest-lib/src/components/KonnectorModal.jsx

```diff
@@ -319,7 +318,6 @@ KonnectorModal.propTypes = {
     })
   }).isRequired,
   findAccount: PropTypes.func.isRequired,
```

Comment:
I think `findAccount ` was injected via `accountsMutation` but I don't see it imported anywhere, did I miss something?

Reply:
😱 Well spotted, I edited https://github.com/cozy/cozy-libs/pull/1021/commits/440c0b1b3e635adfd872df2537e5acbd7368c2da to account for the removal of withMutations.